docs/operator-guide/add-ons-overview.md

```diff
@@ -75,25 +75,38 @@ To enable Add-Ons, it is necessary to have the configured Argo CD, and connect a
 
 Now that Add-Ons are enabled in Argo CD, they can be installed by following the steps below:
 
-1. Choose the Add-On to install.
+1. Decide which Add-Ons you want to install. All the Add-Ons are listed in the [/clusters/core/apps/templates](https://github.com/epam/edp-cluster-add-ons/tree/main/clusters/core/apps/templates) file.
 
-2. On the chosen Add-On, click the **⋮** button and then **Details**:
+2. Enable the Add-Ons chosen in the [/clusters/core/apps/values.yaml](https://github.com/epam/edp-cluster-add-ons/blob/main/clusters/core/apps/values.yaml) file. You can also re-define Add-On parameters. Refer to the example below:
+
+```bash
+kuberocketci-pipelines:
+  enable: true
+  namespace: krci
+  repoUrl: ssh://git@github.com:22/<account-name>/<git-repo-name>.git
+```
+
+:::note
```

Comment:
Its incorrect, in this file you could only enbale addon and select their namespace.
Please delete it